src/w3c/templates/cgbg-headers.js

```diff
@@ -9,11 +9,20 @@ export default conf => {
     <div class="head">
       ${conf.logos.map(showLogo)}
       <h1 class="title p-name" id="title">${conf.title}</h1>
-      ${conf.subtitle
-        ? html`
-            <h2 id="subtitle">${conf.subtitle}</h2>
-          `
-        : ""}
+      ${
```

Comment:
I'm not sure we need this block, as we will end up with two `h1`s (see line 11). Also, it seems we can indeed just rely on `conf.title`.

Reply:
We can just remove the `<h1 class="title p-name" id="title">${conf.title}</h1>` from the code, I guess

Reply:
No, I think we want to keep that first one. All specs need to have a `<h1 class="title...">` so it makes sense just to leave it be.

Reply:
@marcoscaceres We can just go with Line 11 only, I guess and not add the existing block

Reply:
I'll send in the updates code.